packages/vscode-extension/package.nls.cs.json

```diff
@@ -1,584 +1,584 @@
-  "teamstoolkit.walkthroughs.buildIntelligentApps.buildCustomCopilot.title": "Agent vlastního modulu sestavení",
-  "teamstoolkit.walkthroughs.buildIntelligentApps.buildCustomCopilot.description": "Vytvořte robota agenta AI pro běžné úkoly nebo inteligentního chatbota, který odpoví na konkrétní otázky.\n[Build a Basic AI Chatbot](command:fx-extension.createFromWalkthrough?%5B%22WalkThrough%22%2C%20%7B%22capabilities%22%3A%20%22custom-copilot-basic%22%2C%20%22project-type%22%3A%22capabilities%22%0 22custom-copilot-type%22capabilities%22%1 7D%22capabilities%22%2 D)\n[Build an AI Agent](command:fx-extension.createFromWalkthrough?%5B%22WalkThrough%22capabilities%22%5 2C%22capabilities%22%6 7B%22capabilities%22%3A%22capabilities%22%9 22custom-copilot-agent%22%2C%20%22project-type%22%3A%20%22custom-copilot-type%22%7D%5D)\n[Build a Bot to Chat with Your Data](command:fx-extension.createFromWalkthrough?%5B%22WalkThrough%22%2C%20%7B%22capabilities%22%3A%20%22custom-copilot-rag%22%2C%20%22project-type%22%3A%20%22custom-copilot-type%22%7D%5D)",
-  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppResources.title": "Prostředky",
-  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppResources.description": "Prozkoumejte tyto prostředky a vytvářejte inteligentní aplikace a vylepšete své vývojové projekty.\n🗒️ [Generative AI for Beginners](https://github.com/microsoft/generative-ai-for-beginners/tree/main)\n✨ [Načítání rozšířené generace (RAG)](https://learn.microsoft.com/en-us/azure/search/retrieval-augmented-generation-overview)\n📚 [AI Learning and Community Hub](https://learn.microsoft.com/en-us/ai/)",
-  "teamstoolkit.m365.needSignIn.message": "Musíte se přihlásit ke svému účtu Microsoft 365.",
-  "teamstoolkit.handler.createPluginWithManifest.error.missingParameter": "Neplatný parametr v příkazu createPluginWithManifest Použití: createPluginWithManifest(API_SPEC_PATH:string, PLUGIN_MANIFEST_PATH: string, { lastCommand: string }, OUTPUT_FOLDER?: string). Platné hodnoty pro LAST_COMMAND: createPluginWithManifest, createDeclarativeCopilotWithManifest",
-  "teamstoolkit.error.KiotaNotInstalled": "Abyste mohli používat tuto funkci, musíte nainstalovat rozšíření Microsoft Kiota s minimální verzí %s.",
-  "teamstoolkit.handeler.addAuthConfig.notification": "Teams Toolkit has successfully updated your project configuration (teamsapp.yaml and teamsapp.local.yaml) files with added action to support authentication flow. You can proceed to remote provision.",
-  "teamstoolkit.handeler.addAuthConfig.notification.provision": "Provision"
-}
\ No newline at end of file
+{
```

Comment:
Any changes to these localized json files will be overwritten. Please don't modify them.

Reply:
This is caused by merging branch from `dev`